test-requirements.txt

```diff
@@ -28,4 +28,4 @@ testscenarios>=0.4
 testtools>=1.4.0
 mock
 tabulate>=0.7.5
-voluptuous!=0.13.0
+voluptuous!=0.13.0
```

Comment:
Is there a change here? If not, you should be able to remove any changes to this file.